### PR TITLE
(feat) KHP3-6927: Added the discharge workspace UI

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
workspace-discharge

## Related Issue
see:- https://thepalladiumgroup.atlassian.net/browse/KHP3-6927

## Other

*None.*

### CODE REVIEW

File: packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+              className={styles.timePicker}>
+              <TimePickerSelect id="visitStartTimeSelect" labelText={t('time', 'Time')} aria-label={t('time', 'Time')}>
+                <SelectItem value="AM" text="AM" />
+                <SelectItem value="PM" text="PM" />
+              </TimePickerSelect>
+            </TimePicker>
+          </ResponsiveWrapper>
+        </div>
+        <ButtonSet className={styles.buttonSet}>
+          <Button style={{ maxWidth: '57%' }} size="lg" kind="secondary" onClick={closeWorkspace}>
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx:59]: why `57%` here does it scale well for tablets and small view points

File: packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  const layout = useLayoutType();
+
+  return (
+    <Form className={styles.formContainer}>
+      <Stack gap={4} className={styles.formGrid}>
+        <DeceasedInfo patientUuid={patientUuid} />
+        <div className={styles.dateTimeSection}>
+          <DatePicker
+            datePickerType="single"
+            style={{ paddingBottom: '1rem', marginLeft: '1rem' }}
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx:34]: Lets have the styles in `scss` file

**Review by donaldkibet — DISMISSED** [review]

**Comment by its-kios09** [comment]
@donaldkibet I have made adjustments concerning the styling 
discharge

File: packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx (hunk 1 above)
Comment (its-kios09) [reply packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx]: i have removed the styling and moved it to the .scss file

File: packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx (hunk 2 above)
Comment (its-kios09) [reply packages/esm-morgue-app/src/workspaces/discharge-body.workspace.tsx]: sorted

**Comment by its-kios09** [comment]
@makombe i humbly requesting for your review

**Review by makombe — APPROVED** [review]
Thanks @its-kios09

**Pull request merged**